src/src/pages/capabilities/resources/resourceInfoBadges/index.js

```diff
@@ -186,6 +228,77 @@ export function ResourceInfoBadges() {
           </div>
         </>
       )}
+
+      <hr className={styles.divider} />
+
+      <p style={{ textAlign: "center" }}>
+        <img src={azureLogo} alt="Azure icon" style={{ height: "2.5rem" }} />
+      </p>
+
+      <div className={styles.azure}>
+        <div className={styles.items}>
+          {azureResourcesList != [] && isLoadedAzure ? (
+            azureResourcesList.map((x) => (
+              <div key={x.id}>
+                <div className={styles.environment}>
+                  Azure resources for the {x.environment} environment:
+                </div>
+                <div className={styles.azureresource}>
+                  <Badge>
+                    <strong>{x.id} </strong>
+                  </Badge>
+                </div>
+              </div>
+            ))
+          ) : (
+            <div>
+              {" "}
+              <strong>
+                <em style={{ color: theme.colors.text.secondary.primary }}>
+                  No Azure resources linked with this Capability.
+                </em>
+              </strong>
+            </div>
+          )}
+        </div>
+
+        <div className={styles.items}>
+          <div className={styles.envsection}>
+            <div className={styles.envitems}>
+              <label>
+                To create a new Azure resource choose an environment:
+              </label>
+
+              {envAvailability != null ? (
+                <select
+                  style={{ marginLeft: "3px" }}
+                  className={styles.envbutton}
+                  value={environment}
+                  onChange={handleChange}
+                >
+                  {envAvailability.map((env) => (
+                    <option value={env.env} key={env.env} disabled={env.exist}>
+                      {env.env}
+                    </option>
+                  ))}
+                </select>
+              ) : (
+                <>
+                  <div></div>
+                </>
+              )}
+            </div>
+
+            <Button
+              style={{ marginTop: "1rem" }}
+              onClick={() => handleNewAzureResource()}
+            >
+              {" "}
```

Comment:
Pure curiosity; If we wanted more spacing around the text horizontally, why not use something like `padding: 0 4px;` in the already defined styles?

Reply:
{" "} this has to be gone